app/org/sagebionetworks/bridge/models/studies/PasswordPolicy.java

```diff
@@ -0,0 +1,85 @@
+package org.sagebionetworks.bridge.models.studies;
+
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * The policies that govern password creation. New studies will be created with the default password 
+ * policy, which requires 8 characters, one numeric, one symbol and one upper- and lower-case 
+ * character. However, this policy can be adjusted for legacy studies which were created with much 
+ * more lax constraints.
+ */
+public final class PasswordPolicy {
+    
+    public static final int FIXED_MAX_LENGTH = 100;
+    public static final int VALUE_OFF = 0;
+    public static final int VALUE_ON = 1;
+    
+    /**
+     * The password policy that is initially created for a study.
+     */
+    public static final PasswordPolicy DEFAULT_PASSWORD_POLICY = new PasswordPolicy(8, true, true, true);
+    
+    private final int minLength;
+    private final boolean numericRequired;
+    private final boolean symbolRequired;
+    private final boolean upperCaseRequired;
+    
+    @JsonCreator
+    public PasswordPolicy(@JsonProperty("minLength") int minLength,
+                    @JsonProperty("numericRequired") boolean requireNumeric,
+                    @JsonProperty("symbolRequired") boolean requireSymbol,
+                    @JsonProperty("UpperCaseRequired") boolean requireUpperCase) {
+        this.minLength = minLength;
+        this.numericRequired = requireNumeric;
+        this.symbolRequired = requireSymbol;
+        this.upperCaseRequired = requireUpperCase;
+    }
+    
+    public int getMinLength() {
+        return minLength;
+    }
+    public boolean isNumericRequired() {
+        return numericRequired;
+    }
+    public boolean isSymbolRequired() {
+        return symbolRequired;
+    }
```

Comment:
We should document what symbols Stormpath allows/requires. If "symbols" means "any non-alphanumeric char that shows up on a standard QWERTY keyboard", then this is probably fine. Unfortunately, a lot of companies have arbitrary restrictions on symbols, and this causes bad user experience.